private-templates-service/adaptivecards-templating-client/typescript/api.ts

```diff
@@ -14,1487 +14,1343 @@ import localVarRequest = require('request');
 import http = require('http');
 import Promise = require('bluebird');
 
-let defaultBasePath = 'http://localhost:5000';
+let defaultBasePath = 'https://adaptivecms.azurewebsites.net';
```

Comment:
Was this on purpose?

Reply:
Nope, fixed base path in swagger file, thanks for the catch!